docs/config.md

```diff
@@ -55,12 +55,18 @@ for more details.
 
 | ID           | Instrumented library                                                                                                                                                                            | Supported versions | Instrumentation type |
 |--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------------------|
-| `AspNet`     | ASP.NET Framework                                                                                                                                                                               | *                  | source               |
-| `AspNet`     | ASP.NET Core                                                                                                                                                                                    | *                  | source               |
+| `AspNet`     | ASP.NET Framework \[1\]                                                                                                                                                                                    | *                  | source               |
```

Comment:
nit: https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify

Reply:
Thanks for the tip, adding the tool to my VS code.